.github/workflows/test-macos.yml

```
@@ -43,6 +43,13 @@ jobs:
           python-version: ${{ env.PYTHON_VERSION }}
       - name: Environment Information
         run: npx envinfo
+      # This `npm ci` fails a lot as part of the Test step. Run it now so that
+      # we don't have to wait 2 hours for the Build step to pass first before
+      # that failure happens. (And if there's something about `make run-ci -j2`
+      # that is causing the failure and the failure doesn't happen anymore if
+      # we run `npm ci` here first, that's also useful information.)
+      - name: npm ci workaround
+        run: (cd tools/doc && npm ci)
```

Maybe this would be better?

```suggestion
        run: make tools/doc/node_modules
```

---

This might address the concern in https://github.com/nodejs/node/pull/41035#issuecomment-982717008?